contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -467,20 +643,65 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
         address _recipient,
         address _asset,
         uint256 _amount
-    ) external virtual override {
+    ) 
+        external
+        override
+        onlyVault
+        nonReentrant
+    {
+        require(_asset == WETH, "Unsupported asset");
```

Comment:
If this is only allowing withdraws to the vault, it should revert if _recipient is ever a non vault address.

Reply:
yeah true, this will never be a strategy that will withdraw anywhere else than to the vault.

Reply:
fixed here: https://github.com/OriginProtocol/origin-dollar/pull/2184/commits/a6e6ca721b817187694d65a793c04e7760a02666